client/src/components/Card.tsx

```
@@ -9,7 +10,7 @@ interface CardProps {
 const Card: React.FC<CardProps> = ({ imageSrc, title, description }) => {
   return (
     <div className="flex flex-col items-center justify-center border border-gray-300 p-6 basis-full md:h-80 rounded-md shadow-lg hover:shadow-xl hover:bg-pink-100 bg-gray-50">
-      <img src={imageSrc} alt="Card Image" className="w-24 h-24 mb-4" />
+      <img src={imageSrc}  className="w-24 h-24 mb-4" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)